experiments/shield_study_engineering.md

```diff
@@ -0,0 +1,134 @@
+# Shield Study Engineering
+
+**NOTE**:  this is a work in progress.  Feedback and questions welcome.
+
+## Code is Expensive.
+
+You want to write code.   Fight that urge.
+
+**Shield Study Addons are very expensive, in both time and engineering cost**.  
+
+Are you sure you [need a Shield Study](./need_shield_study.md)?  
+
+If not, [What Design Do I Need?](experiments/shield_design_process.md)
+
+You are an expert on coding.  We are experts on experiment design.
+In particular, can you get your answers with a Survey?  If so, do that.  Have you gone through the design wizard?
+
+**Important**
+
+0. Are you coding first, before you have a decision to make / evidence collection plan?  If so, please stop and [contact us](./shield_help.md).
+
+1. If we don't know about your project, [contact us](./shield_help.md). We cannot launch or offer engineering support to projects we don't know about.  
+
+2. If your project doesn't have prior research from Surveys or other methods, we will not ship it.  [contact us](./shield_help.md).  
+
+
+## What Is A Shield Study?
+
+A Shield Study is an instrumented addon.  
+
+A Shield Study is built and designed to expose users to probes to answer questions.  
+
+All engineering and process decisions in the Shield Program serve this goal:  **get correct probes to users**.    
+
+
+## Differences from General Feature Coding
+
+You are used to making robust, high-quality Features. 
+
+If you attempt to build a production level feature **and instrument it**, it will be more expensive than landing.
+
+1.  In Study Engineering, the main goal is code to make probes survive the wild.  The Feature is secondary.  
+2. QA and Engineering acceptance needs to check that probability.
+3. Make the feature as thing as you can.  Some techniques:
+
+
+## Topics of Shield Engineering
+
+
+### Do I need the Study Utils?
+
+[shield-studies-addon-utils](https://github.com/mozilla/shield-study-addon-utils)
+
+Study Utils makes these things easier.
```

Comment:
Give some examples of mishaps when people have NOT used the utils. For example: There was a recent study that didn't [`setExperimentActive`](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/experiments.html?highlight=setexperimentactive#the-js-api), and so the analyst couldn't tell which users were actually enrolled in the experiment.